packages/vgplot/src/spec/parse-spec.js

```diff
@@ -363,8 +364,9 @@ function parseInteractor(spec, ctx) {
   return fn(options);
 }
 
-function parseExpression(spec, ctx) {
-  const { expr, label } = spec;
+function parseExprOrAgg(spec, ctx, key, func) {
```

Comment:
As a simpler alternative, perhaps we can instead leave `parseExpression` as it was, and add a new `parseAggregation` method like so:

```js
function parseAggregation(value, this) {
  return parseExpression(value, this).annotate({ aggregate: true });
}
```

We can then drop the `agg` import as well.

Reply:
Note that parseExpression still needs the key ("expr" or "agg") so it can't be kept literally as is (but I implemented something like your suggestion in to-module.js adding the default kwarg)